axon-server-connector/src/main/java/org/axonframework/axonserver/connector/command/AxonServerRemoteCommandHandlingException.java

```diff
@@ -38,7 +39,18 @@
      * @param errorMessage the message describing the exception on the remote end
      */
     public AxonServerRemoteCommandHandlingException(String errorCode, ErrorMessage errorMessage) {
-        super(new RemoteExceptionDescription(errorMessage.getDetailsList()));
+        this(errorCode, errorMessage, false);
+    }
+
+    /**
+     * Initialize the exception with given {@code errorCode}, {@code errorMessage} and {@code writableStackTrace}.
+     *
+     * @param errorCode          the code reported by the server
+     * @param errorMessage       the message describing the exception on the remote end
+     * @param writableStackTrace whether the stack trace should be generated ({@code true}) or not ({@code false}, this is a default)
```

Comment:
For this constructor, the `writableStackTrace` parameter has no default. 
Hence, I think the statement `({@code false}, this is a default)`  shouldn't belong in this piece of JavaDoc. 
I'd wager it is better situated in the other constructor, as that actually defines the default.

Reply:
Similarly, as with the class level JavaDoc comment, this comment can also be copy-pasted to every file that makes the same adjustment.